packages/presentation/presentation_publishing/comparators/fallback_comparator.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import { BehaviorSubject } from 'rxjs';
+import { PublishingSubject } from '../publishing_subject';
+import { ComparatorDefinition } from './types';
+
+/**
+ * Comparators are required for every runtime state key. Occasionally, a comparator may
+ * actually be optional. In those cases, implementors can fall back to this blank definition.
+ */
+export const getFallbackComparator = <
```

Comment:
How about renaming to "getNeverChangesComparator" or something like that?

"getFallbackComparator" is too vague and it could be mis understood as a default implementation, when it really never compares anything.

Reply:
Renamed to `getUnchangingComparator` in [e385b87](https://github.com/elastic/kibana/pull/181751/commits/e385b871ae7e03dee486d27a655f93ac35dd1297)